vbfml/scripts/preprocess_image.py

```diff
@@ -295,16 +295,18 @@ def check_met(input_file: str, name_save: str, start: int, stop: int):
     batch_counter = 0
 
     # write MET distribution in .pkl file
-    cache = pjoin(
-        output_dir, f"{os.path.basename(input_file)[5:-5]}_met_distribution.pkl"
+    dataset_name = (
+        os.path.basename(input_file).replace("tree_", "").replace(".root", "")
```

Comment:
good to know we can do that 
I didn't find any other way so thank you for this upgrade